helloworld/Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/groovy
+node('redhatdistortion-maven') {
+    stage 'Checkout'
+    checkout scm
+
+    stage 'Build'
+    sh 'cd helloworld && mvn clean compile'
+
+    stage 'Run Unit Tests'
+    sh 'cd helloworld && mvn test'
+
+    stage 'Package'
+    sh 'cd helloworld && mvn package'
+
+    stage 'Archive artifact'
+    archive 'helloworld/target/*.war'
+
+    stage 'Create Image'
+    sh 'oc login https://kubernetes.default/ -u openshift-dev -p devel --insecure-skip-tls-verify=true'
+    sh 'oc project helloworld-dev || oc new-project helloworld-dev'
+    sh 'oc process -f https://github.com/tnozicka/openshift-templates/raw/master/binary-build-template.yaml ' +
+       '-v "APP_NAME=helloworld,BUILDER_IMAGE=registry.access.redhat.com/jboss-eap-7/eap70-openshift,OUTPUT_IMAGESTREAM_TAG=helloworld:latest" ' +
+       '| oc apply -f -'
+    sh 'mkdir -p ../artifacts/deployments/ && cp helloworld/target/*.war ../artifacts/deployments/'
+    sh 'oc delete builds -l buildconfig=helloworld'
```

Comment:
Deleting all previous builds seems a bit over the top. Shouldn't previous builds be kept?